utils/api_requests.py

```
@@ -254,10 +254,10 @@ def get_raidbots_item_squish_era():
             return {}
 
 
-def get_ilvl_items(ilvl=201, item_ids=[]):
+def get_ilvl_items(ilvl=196, item_ids=[]):
```

Ruff (0.15.6)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)